src/pages/example--gift-card.mdx

```diff
@@ -0,0 +1,1120 @@
+import { Callout } from "nextra-theme-docs";
+
+# Gift Card
+
+Let's build a UI to send and redeem a gift card using smart contracts on Cardano.
+
+## Covered in this tutorial
+
+- writing `Aiken`
+- parameterizing scripts
+- using [lucid](https://lucid.spacebudz.io/) (requires a [blockfrost](https://blockfrost.io/) account)
```

Comment:
Just a thought: we could also show a setup using Kupo+Ogmios via Demeter.

Reply:
I'm down for that

Reply:
done!